ocfweb/docs/docs/staff/procedures/new-host.md

```diff
@@ -138,3 +138,30 @@ until the configuration converges.
 The first Puppet run and various other things may be broken if one or more
 packages are out of date, e.g. Puppet. Remedy this with an `apt-get update &&
 apt-get upgrade`.
+
+## Tips and Tricks
+
+### VNC monitoring of Debian install
+
+In some cases it may be useful to monitor a VM install via VNC, if, for example,
+the install is getting stuck somewhere. To do so, run the following.
+
+First, start the virsh VNC server on the target hypervisor:
+
+    $ sudo virsh vncdisplay <vm name>
+    127.0.0.1:5
+
+The number after the colon, + 5900, is the port the VNC server is listening on. In
+this case, the VNC server is listening on `<hypervisor>:5905`.
+
+Next, open a new terminal on a desktop and open an SSH tunnel to the hypervisor on the
+port the VNC is listening on, via, e.g.:
+
+    ssh -L 5900:localhost:5905 <hypervisor>
+
+Finally, in yet another terminal, launch a VNC display to view the console by doing:
+
+    vncviewer localhost:5900
+
+This will open a VNC window that will allow you to monitor and interact with the
+remote console.
```

Comment:
These instructions seem really complicated, can't you just replace this section with these two lines?

```
me@desktop:~$ ssh -X jaws
me@jaws:~$ sudo -E virt-viewer death

Reply:
nice, done